### PR TITLE
[feature] Export StopPlace for NeTEx France

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

**Comment by woshilapin** [comment]
We found some inconsistencies with NeTEx norm. `Quay` can't refer to `StopPlace` with `ParentSiteRef`. We need to create a link from `StopPlace/quays/QuayRef[]` towards `Quay` elements. Therefore, I'm putting the PR in `wip`.

**Comment by woshilapin** [comment]
I removed `wip` tag from this PR.

File: tests/fixtures/netex_france/arrets.xml [hunk 1]
```diff
@@ -3,159 +3,399 @@
 	<PublicationTimestamp>2019-04-03T17:19:00</PublicationTimestamp>
 	<ParticipantRef>Participant</ParticipantRef>
 	<dataObjects>
-		<GeneralFrame>
+		<GeneralFrame id="FR:GeneralFrame:NETEX_ARRET:ProviderCode" version="any">
```
Comment (ArnaudOggy) [inline tests/fixtures/netex_france/arrets.xml]: <CompositeFrame id="FR:CompositeFrame:NETEX_ARRET:ProviderCode" version="any">
  <frames>
    <ServiceFrame id="FR:GeneralFrame:NETEX_ARRET:ProviderCode" version="any">
@prhod 
CompositeFrame or no to be iso with the lignes.xml file (and netex_idf) ?
I know that the 2 options are valid netex 1.9. It tickles me a little without :smile:

File: src/documentation/ntfs_to_netex_france_specs.md [hunk 2]
```diff
@@ -144,12 +144,11 @@ Netex field | NTFS file | NTFS field | Note
 --- | --- | --- | ---
 Quay/@id | stops.txt | stop_id | see the section [Identifiers for Quay and StopPlace](#identifiers-for-quay-and-stopplace)
 Quay/@version | | | fixed value `any`.
-Quay/PublicCode | stops.txt | stop_code | This node may not be present if the stop_point has no `stop_code`.
-Quay/Name | stops.txt | stop_name | 
+Quay/Name | stops.txt | stop_name |
 Quay/Centroid/Location | stops.txt | stop_lat and stop_lon | see [Coordinates conversion](#coordinates-conversion)
-Quay/ParentSiteRef | stops.txt | parent_station | see the section [Identifiers for Quay and StopPlace](#identifiers-for-quay-and-stopplace)
 Quay/TransportMode | | | see (2) below
 Quay/tariffZones | stops.txt | fare_zone_id | The fare zone is prefixed by the `ParticipantRef` prefix with a `:` separator
```
Comment (ArnaudOggy) [inline src/documentation/ntfs_to_netex_france_specs.md]: Quay/tariffZones/TariffZoneRef/@ref

Comment (ArnaudOggy) [inline src/documentation/ntfs_to_netex_france_specs.md]: Quay/tariffZones/TariffZoneRef/@ref

Comment (woshilapin) [reply src/documentation/ntfs_to_netex_france_specs.md]: Thanks, done in 7b015b04663426e66cc7a63d0e9a5735364defb6.

**Comment by woshilapin** [comment]
@ArnaudOggy I pushed [this commit](https://github.com/CanalTP/transit_model/commit/800f2a76aae2a623e6f46fe689afd2ec33c20275) to avoid creations of `Quay` and `StopPlace` when a specific NeTEx mode is not supported by our exporter.

**Pull request merged**